Stap Dan heeft {aanduiding} geen gezaghouder heeft onjuiste verwachting

Als ik een scenario uitvoer met de stap Dan heeft 'Bert' geen gezaghouder met profiel GezagApiDeprecated dan is de verwachting niet correct.

```
    Voorbeeld: De persoon is meerderjarig
      Gegeven persoon 'Bert'
      * is meerderjarig
      Als 'gezag' wordt gevraagd van 'Bert'
      Dan heeft 'Bert' geen gezaghouder
```
geeft bij uitvoeren met -p GezagApiDeprecated:
```
       AssertionError: actual: {
       	"personen": [
       		{
       			"burgerservicenummer": "000000036",
       			"gezag": []
       		}
       	]
       }
       expected: {
       	"personen": [
       		{
       			"gezag": []
       		}
       	]
       }
```

N.B. bij uitvoeren met -p InfoApiDeprecated klopt de verwachting wel

--- a/features/docs/dan-stap-definities-gezagsrelatie-expressief.feature
+++ b/features/docs/dan-stap-definities-gezagsrelatie-expressief.feature
@@ -559,6 +559,7 @@ Functionaliteit: Stap definities ten behoeve van specificeren gezagsrelaties
 
   Regel: Dan heeft '{aanduidingMinderjarige}' geen gezaghouder
 
+    @info-api @data-api
     Scenario: gevraagde persoon heeft geen gezagsrelaties
       Gegeven de response body is gelijk aan
         """
@@ -572,8 +573,23 @@ Functionaliteit: Stap definities ten behoeve van specificeren gezagsrelaties
         """
       Dan heeft 'Bert' geen gezaghouder
 
+    @gezag-api
+    Scenario: gevraagde persoon heeft geen gezagsrelaties
+      Gegeven de response body is gelijk aan
+        """
+        {
+          "personen": [
+            {
+              "burgerservicenummer": "000000036",
+              "gezag": []
+            }
+          ]
+        }
+        """
+      Dan heeft 'Bert' geen gezaghouder
+
   Regel: De response kan meerdere gezagsrelaties bevatten
-    
+
     @deprecated @gezag-api
     Scenario: meerdere gezagsrelaties van één persoon
       Gegeven de response body is gelijk aan

--- a/features/step_definitions/dan-stepdefs-gezagsrelatie.js
+++ b/features/step_definitions/dan-stepdefs-gezagsrelatie.js
@@ -239,6 +239,10 @@ Then('heeft {aanduiding} geen gezaghouder', function (aanduidingMinderjarige) {
         ]
     };
 
+    if(this.context.isGezagApiAanroep) {
+        expected.personen[0].burgerservicenummer = getBsn(getPersoon(this.context, aanduidingMinderjarige));
+    }
+
     this.context.expected = expected;
 });
 

